docs/en/guides/model-deployment-options.md

```diff
@@ -258,6 +258,10 @@ NCNN is a high-performance neural network inference framework optimized for the
 
 - **Hardware Acceleration**: Tailored for ARM CPUs and GPUs, with specific optimizations for these architectures.
 
+#### MNN
+
+MNN is a highly efficient and lightweight deep learning framework. It supports inference and training of deep learning models and has industry-leading performance for inference and training on-device. In addition, MNN is also used on embedded devices, such as IoT.
+
 ## Comparative Analysis of YOLO11 Deployment Options
```

Comment:
@wangzhaode Can we have the MNN in the following table as well?

Reply:
![zHqqLhpo1I](https://github.com/user-attachments/assets/918ae31a-6b2f-4dd2-9cdd-9d637ba28efc)

Reply:
Yeah, added in [edf0a77](https://github.com/ultralytics/ultralytics/pull/16802/commits/edf0a77d718ac857b7af84a2d14e442bbd416b17)